yt/yt/tests/integration/misc/test_logging.py

```diff
@@ -0,0 +1,166 @@
+from yt_env_setup import YTEnvSetup
+
+from yt_commands import authors, create, sync_mount_table, sync_create_cells, select_rows, start_transaction, set
+
+import time
+
+##################################################################
+
+
+class TestDynamicTableLogger(YTEnvSetup):
+    NUM_MASTERS = 3
+    NUM_NODES = 1
+    USE_DYNAMIC_TABLES = True
+
+    DEBUG_LOG_TABLE = "//tmp/debug_log"
+    ACCESS_LOG_TABLE = "//tmp/access_log"
+
+    DELTA_MASTER_CONFIG = {
+        "logging": {
+            "flush_period": 100,
+            "shutdown_busy_timeout": 1000,
+            "rules": [
+                {
+                    "family": "plain_text",
+                    "min_level": "debug",
+                    "exclude_categories": ["Bus"],
```

Comment:
We need the same fix here that was made for the trunk branch.

Reply:
Done.